src/components/chemistry/pubchem/computed/Computed.tsx

```diff
@@ -7,34 +7,46 @@ export default function Computed(props: { computed: any }) {
   const rows: Array<any> = [];
   for (let key in computed) {
     rows.push({
-      label: key,
+      key: key,
+      label: computed[key].label,
       value:
         computed[key].value +
         (computed[key].units ? ` ${computed[key].units}` : ''),
-      description: computed[key].reference.description,
+      reference: computed[key].reference.description,
+      description: computed[key].description,
     });
   }
 
-  return <Table Header={Header} data={rows} Tr={Row} />;
+  return (
+    <Table
+      Header={Header}
+      data={rows}
+      Tr={Row}
+      tableClassName="table-fixed"
+      tableStyle={{ width: '50%' }}
+    />
+  );
 }
 
 function Header() {
   return (
-    <tr>
-      <Th>Label</Th>
+    <tr key="computedPropertiesHeader">
```

Comment:
```suggestion
    <tr>
```

In React, keys must be used when arrays of elements are returned or in rare cases to force React to completely re-render a component subtree. This key doesn't really have any effect.

https://reactjs.org/docs/lists-and-keys.html#keys